README.md

```diff
@@ -65,10 +65,7 @@ package in the image.
 
 #### Accounting
 
-By default, the accounting plugin will use the `accounting_storage/filetxt` storage type. However,
-this only supports a subset of `sacct` commands.
-
-To deploy and configure `slurmdbd`:
+By default, no accounting storage is configured. OpenHPC v1.x and un-updated OpenHPC v2.0 clusters support file-based accounting storage which can be selected by setting the role variable `openhpc_slurm_accounting_storage_type` to `accounting_storage/filetxt`<sup id="accounting_storage">[1](#slurm_ver_footnote)</sup>. Accounting for OpenHPC v2.1 and updated OpenHPC v2.0 clusters requires the Slurm database daemon, `slurmdbd`. To enable this:
```

Comment:
Hmm it might be a matter of personal preference but footnote in markdown seems to make it less readable.